Resources/Views/Hub/home.leaf

```diff
@@ -0,0 +1,330 @@
+                        <input type="text" id="fn" class="form-control border-0 bg-transparent shadow-none" placeholder="Enter your YouTube channel URL" readonly>
+                        <div class="p-2">
+                            <i class="bx bxl-youtube fs-xl opacity-60 me-2 mt-1"></i>
+                        </div>
+                      </div>
+                  </div>
+                </div>
+                #if(false):
+                <div class="d-flex mb-3">
+                  <button type="reset" class="btn btn-secondary me-3">Cancel</button>
+                  <button type="submit" class="btn btn-primary">Save changes</button>
+                </div>
+                #endif
+              </form>
+#endif
+
+#if(false):
+              <!-- Merch -->
+              <h2 id="merch" class="h5 text-primary pt-1 pt-lg-3 mt-4">Merch Preferences</h2>
+              
+                <div class="form-check form-switch">
+                  <input type="checkbox" class="form-check-input" id="customSwitch2" checked>
+                  <label class="form-check-label" for="customSwitch2">Do you want SwiftLeeds Swag?</label>
+                </div>
+                
+                <div class="range-slider" data-start-min="450" data-min="0" data-max="10" data-step="1" data-tooltip-prefix="$">
+                  <div class="range-slider-ui"></div>
+                  <input class="form-control range-slider-value-min" type="hidden">
+                </div>
+              
+              <hr />
+#endif
+              
+            #if(count(dropInSessions) > 0):
+              <!-- Drop-in -->
+              <h2 id="dropin" class="h5 text-primary pt-1 pt-lg-3 mt-4">Drop-in Sessions</h2>
+              <p>Drop-in sessions enable you to have a one-to-one conversation with one of our fantastic hosts below. Each cover a unique subject, covering your apps, career or anything else you want to discuss.</p>
+              <p>Drop-in sessions are entirely free, with one session included in your ticket cost. Slots are limited and are offered on a first come first served basis, so why not book a slot below?</p>
+              
+              #if(prompt):
+                <div class="alert alert-secondary" role="alert">#(prompt)</div>
+              #endif
+              
+              #for(session in dropInSessions):
+              
+                <div class="d-flex flex-row align-items-center mt-5 mb-3">
+                  <img src="#awsImage(session.ownerImageUrl)" class="d-block rounded" width="80" alt="#(session.ownerName)">
+                  <div class="ps-4">
+                    <h5 class="fw-medium fs-lg mb-1">#(session.title)</h5>
+                    <p class="fs-sm mb-0">#(session.ownerName)</p>
+                  </div>
+                </div>
+              
+              <div class="row flex-nowrap overflow-auto">
+              #for(slot in session.slots):
+                <div class="col-xs-5 col-md-5 col-sm-5 col-lg-4 col-xl-3">
```

Comment:
Matts iPhone doesn't seem to be respecting these, might need to add col-5 class too for extra small devices. Do a responsive test